pom.xml

```diff
@@ -135,6 +135,11 @@
       <classifier>tests</classifier>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.jenkins-ci.plugins</groupId>
+      <artifactId>handy-uri-templates-2-api</artifactId>
+      <version>2.1.6-1.0-20180102.164938-1</version>
```

Comment:
Merging blocked until this is released

Reply:
it just got hosted, so I'll upload it tonight 👍